widget/embedded/src/types/notification.ts

```diff
@@ -21,5 +21,6 @@ type NotificationRoute = {
 export type Notification = Pick<Route, 'requestId'> & {
   event: RouteEvent | StepEvent;
   creationTime: number;
+  transactionCreationTime: number;
```

Comment:
I believe it is more related to the route information, so it would be better to move this property to the route object:
```ts
type NotificationRoute = {
  transactionCreationTime: number;
  from: {
    blockchain: Step['fromBlockchain'];
    symbol: Step['fromSymbol'];
    address: Step['fromSymbolAddress'];
  };
  to: {
    blockchain: Step['toBlockchain'];
    symbol: Step['toSymbol'];
    address: Step['toSymbolAddress'];
  };
};
```

Reply:
Tnx Fixed